src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -437,6 +437,7 @@ private static boolean isAnonymousClassDef(DetailAST ast) {
      * or null otherwise.
      * @param ast IDENT ast to check.
      * @return the class frame where violation is found or null otherwise.
+     * @noinspection IfStatementWithIdenticalBranches
```

Comment:
wercker message "Command timed out after no response" 
Execution is hanged on 
`[INFO] Generating "Checkstyle" report    --- maven-checkstyle-plugin:2.17`
So Check is hanging during execution over checkstyle and sevntu.checkstyle sources.
Please reproduce it on your local